layouts/partials/sidebar.html

```diff
@@ -1,9 +1,9 @@
 {{ $currentPage := . }}
-<div class="relative w-20-ns ph4-l ph3 flex justify-end-ns br bg-near-white b--light-gray">
+<div id="sidebar" class="relative ph4-l ph3 flex justify-end-ns br bg-near-white b--light-gray">
```

Comment:
added ID for custom CSS since the largest tachyon class added only `16rem` of width, and was not sufficient to avoid the squish